frontend/pages/index.tsx

```diff
@@ -32,29 +33,23 @@ export default function HomePage() {
           alignItems="flex-start"
         >
           <Grid item xs={12} sm={12} md={7} lg={9}>
-            <Link href="/news">
+            <NextLink href="/news">
```

Comment:
In some files this is import is named NextLink and in others it is Link while mui is renamed to MuiLink

Reply:
I didn't get an action here but I will rename the one "NextLink" import to "Link" and if we use both use "Link" and "MuiLink". It is the name of the component after all [https://mui.com/api/link/#component-name](https://mui.com/api/link/#component-name)